contivctl

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+VERSION=0.0.1
+URL=http://contiv.io 
+
+showhelp() {
+    cat 1>&2 <<EOF
+contivctl - manage contiv control plane
+
+VERSION: ${VERSION}
+
+Commands:
+
+  - network <arguments>: control contiv network
+  - storage <arguments>: control contiv storage
+  - cluster <arguments>: control contiv cluster
+EOF
+    exit 1
+}
+
+missingcommands() {
+  cat 1>&2 <<EOF
+It looks like you're missing the contiv distribution! Please visit
+${URL} for more information on how to install it.
+EOF
+  exit 2
+}
+
+check() {
+  for cmd in netctl volcli clusterctl
+  do
+    if !which $cmd &>/dev/null
+    then
+      missingcommands
+      exit 1
+    fi
+  done
+}
+
+if [ "x$1" = "x" ]
+then
+  showhelp
+fi
+
+check
```

Comment:
ok, I'll do that in a little bit here.

On 11 Apr 2016, at 12:17, Madhav Puri wrote:

>> +  for cmd in netctl volcli clusterctl
>> +  do
>> +    if !which $cmd &>/dev/null
>> +    then
>> +      missingcommands
>> +      exit 1
>> +    fi
>> +  done
>> +}
>> +
>> +if [ "x$1" = "x" ]
>> +then
>> +  showhelp
>> +fi
>> +
>> +check
>
> may be we should do the check only when a particular sub-command is 
> called because there might be nodes where we may have installed 
> netplugin but not volplugin (for networking only setups) and 
> vicee-versa, same will hold true for cluster-manager as it is not 
> installed on all nodes in a cluster.
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/contivctl/pull/2/files/c107edeb21e402f9f9db759447b43f6d1466b24d#r59263875